src/js/InvertedIndex.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * InvertedIndex class
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.files = [];
+  }
+
+  /**
+   * Validates file content
+   * @function
+   * @param {Array} fileContent content of uploaded file
+   * @return {Array|Boolean} valid fileContent or false if invalid file
+   */
+  static validateFile(fileContent) {
+    if (!(fileContent instanceof Array) || fileContent.length < 1) {
```

Comment:
What exactly are you trying to achieve here, shouldn't your file content be an object?